miden-tx/src/tests/kernel_tests/test_faucet.rs

```diff
@@ -49,16 +49,16 @@ fn test_mint_fungible_asset_succeeds() {
             assert_eqw
 
             # assert the input vault has been updated
-            exec.memory::get_input_vault_root_ptr
-            push.{ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN}
-            exec.asset_vault::get_balance
-            push.{FUNGIBLE_ASSET_AMOUNT} assert_eq
+            #exec.memory::get_input_vault_root_ptr
+            #push.{ACCOUNT_ID_FUNGIBLE_FAUCET_ON_CHAIN}
+            #exec.asset_vault::get_balance
+            #push.{FUNGIBLE_ASSET_AMOUNT} assert_eq
 
             # assert the faucet storage has been updated
-            push.{FAUCET_STORAGE_DATA_SLOT}
-            exec.account::get_item
-            push.{expected_final_storage_amount}
-            assert_eq
+            #push.{FAUCET_STORAGE_DATA_SLOT}
+            #exec.account::get_item
+            #push.{expected_final_storage_amount}
+            #assert_eq
```

Comment:
This should probably be un-commented?

Reply:
Indeed, I forgot to work on this. The test also passes when un-commented.